docs/TESTING.md

```
@@ -0,0 +1,58 @@
+- **Axe Core**: Automated accessibility audits.
+
+## Running Tests
+
+### Unit & Integration (Vitest)
+
+```bash
+npm test
+```
+
+To run in watch mode:
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)